docs/development/l10n.rst

```diff
@@ -293,6 +293,32 @@ like Google, Paypal and Twitter now offer two-factor authentication.
 
 This definition was copied from `the EFF glossary <https://ssd.eff.org/en/glossary/two-factor-authentication>`__
 
+Weblate glossary
+----------------
+
+For each string to be translated, Weblate shows a glossary of terms
+and their translation to help unify their translations. For instance
+when translating `Please wait for a new two-factor token before
+logging in again`, Weblate notices the word `two-factor` is found in
+the glossary and displays the translation in the `glossary` to the
+right.
+
+|Weblate glossary show page screenshot|
+
+Before translating strings, it is recommended to add all term in the
+`SecureDrop localization glossary <#glossary>`_ by clicking on the pen
+in the right corner of the glossary displayed with each translated
+string and then `Add new word`:
+
+|Weblate glossary add page screenshot|
+
+When done review and fix the glossary.
```

Comment:
What do you mean by fix the glossary here?

Reply:
I updated that (too) terse sentence with:

<pre>
When all the terms are in the glossary, it is recommended to take
another look at the full list of terms and verify there is no
duplicate or other mistakes.
</pre>